prjxray/fasm_assembler.py

```diff
@@ -0,0 +1,152 @@
+import fasm
+from prjxray import bitstream
+from prjxray import grid
+
+
+class FasmLookupError(Exception):
+    pass
+
+
+class FasmInconsistentBits(Exception):
+    pass
+
+
+# How many 32-bit words for frame in a 7-series bitstream?
+FRAME_WORD_COUNT = 101
```

Comment:
we should probably move this constant to a central location

Reply:
Ya, I moved the word size constant, I'll move this too.